From 989bc886466076d452914d746c150eb0ae91c9c7 Mon Sep 17 00:00:00 2001
From: Simon Symeonidis <lethaljellybean@gmail.com>
Date: Wed, 7 May 2025 19:36:22 -0400
Subject: [PATCH] Change log.Println statement to log.Printf

This changes a statement from Println, to Printf, as it seems the
original intention was to format the error.
---
 ui/browsing/trackspage.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ui/browsing/trackspage.go b/ui/browsing/trackspage.go
index eaf34563..3e42edc9 100644
--- a/ui/browsing/trackspage.go
+++ b/ui/browsing/trackspage.go
@@ -210,7 +210,7 @@ func (t *TracksPage) playRandomSongs() {
 	go func() {
 		err := t.contr.App.PlaybackManager.PlayRandomSongs("")
 		if err != nil {
-			log.Println("error playing random tracks: %v", err)
+			log.Printf("error playing random tracks: %v", err)
 			fyne.Do(func() {
 				t.contr.ToastProvider.ShowErrorToast(lang.L("Unable to play random tracks"))
 			})
